### PR TITLE
Adding Bootstrap Assignment 5&6

### PR DESCRIPTION
**Pull request by somesh6797**

### CODE REVIEW

File: Somesh/JavaScript/Assignment1/que1Assign1.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                padding: 10px;
+                border-radius: 8px;
+                box-shadow: 0px 0px 4px rgb(0, 0, 0),
+                    inset 0px 0px 8px rgb(0, 0, 0);
+                display: flex;
+                flex-direction: column;
+                justify-content: flex-start;
+                align-items: center;
+            }
+
```
Comment (sourabh-kanwade) [inline Somesh/JavaScript/Assignment1/que1Assign1.html]: remove space

File: Somesh/JavaScript/Assignment1/que2Assign1.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                padding: 10px;
+                border-radius: 8px;
+                box-shadow: 0px 0px 4px rgb(0, 0, 0),
+                    inset 0px 0px 8px rgb(0, 0, 0);
+                display: flex;
+                flex-direction: column;
+                justify-content: flex-start;
+                align-items: center;
+            }
+
```
Comment (sourabh-kanwade) [inline Somesh/JavaScript/Assignment1/que2Assign1.html]: remove space

File: Somesh/JavaScript/Assignment1/que2Assign1.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                font-size: 14px;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <input type="text" placeholder="temperature in celcius" id="temp" />
+
+            <button onclick="convert()">To Fahrenheit</button>
+
```
Comment (sourabh-kanwade) [inline Somesh/JavaScript/Assignment1/que2Assign1.html]: remove space

File: Somesh/JavaScript/Assignment1/que2Assign1.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                border: 2px solid white;
+                color: white;
+                font-size: 14px;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <input type="text" placeholder="temperature in celcius" id="temp" />
+
```
Comment (sourabh-kanwade) [inline Somesh/JavaScript/Assignment1/que2Assign1.html]: remove space

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]
Remove space

**Review by Pawan1005 — APPROVED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make changes

**Review by sourabh-kanwade — APPROVED** [review]

**Review by kalyani1942 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**